### DISCUSSION TITLE
Read key shortcut input (Command) on OSX

### DISCUSSION THREAD

**Discussion by hoffenmerCode** [beeware/toga#1299] [Q&A]
Couldn't find a working example of this. Can someone give me a code example. 
Trying to invoke a function when Command+B is pressed in my main window. 
How would I do that?

**Comment by freakboy3742** [comment]
[Tutorial 2](https://github.com/beeware/toga/blob/master/examples/tutorial2/tutorial/app.py) contains a demonstration of this. You define a Command, and bind that command to the app.
